ReactNative/ReactTwitter/core/views/tweetView.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import {
+  StyleSheet,
+  View,
+  Image,
+  Text,
+  BackAndroid,
+  Navigator
+} from 'react-native'
+
+var dateFormat = require('dateformat')
+
+var TweetsView = React.createClass({
+  propTypes: {
+    tweetId: React.PropTypes.string.isRequired,
+    navigator: React.PropTypes.instanceOf(Navigator).isRequired
+  },
+
+  getInitialState () {
+    return {
+      tweet: {}
+    }
+  },
+
+  componentDidMount () {
+    BackAndroid.addEventListener('hardwareBackPress', () => {
```

Comment:
we probably need this behaviour on different Screen, Is it worth to create a super-class? Or some kind of mixin?